lib/pensio_api/terminal.rb

```diff
@@ -9,7 +9,7 @@ class Terminal
     attr_reader :terminal_methods
 
     def self.all
-      request = Request.new('/merchant/API/getTerminals')
+      request = PensioAPI::Request.new('/merchant/API/getTerminals', method: 'GET')
```

Comment:
PensioAPI:: wasn't enforced previously, is it required?

Reply:
No, it is not required. The code works without it. I have updated the code.